extractor/src/main/java/dev/_00a/valence_extractor/extractors/Items.java

```diff
@@ -0,0 +1,33 @@
+package dev._00a.valence_extractor.extractors;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import dev._00a.valence_extractor.Main;
+import net.minecraft.util.registry.Registry;
+
+public class Items implements Main.Extractor {
+    public Items() {
+    }
+    @Override
+    public String fileName() {
+        return "items.json";
+    }
+
+    @Override
+    public JsonElement extract() throws Exception {
+        var itemsJson = new JsonArray();
+
+        for (var item : Registry.ITEM) {
+            var itemJson = new JsonObject();
+            itemJson.addProperty("id", Registry.ITEM.getRawId(item));
+            itemJson.addProperty("name", Registry.ITEM.getId(item).getPath());
+            itemJson.addProperty("translation_key", item.getTranslationKey());
+            itemJson.addProperty("max_stack", item.getMaxCount());
+
```

Comment:
Is this everything we need to be able to convert items to block states? Go ahead and grab as much info as you can.